frontend/src/pages/lesson/Activity.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import DoneRoundedIcon from '@material-ui/icons/DoneRounded';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import { DataGrid } from '@material-ui/data-grid';
+import { useHistory } from 'react-router-dom';
+import PageContainer from '../_shared/PageContainer';
+import Title from '../_shared/Title';
+
+const convertDateToReadableString = (date) => {
+  return `${date.substring(0, 10)} ${date.substring(11, 19)}`;
+};
+
+const columns = [
+  {
+    field: 'updatedAt',
+    headerName: 'Datum',
+    flex: 0.2,
+  },
+  {
+    field: 'student',
+    headerName: 'Leerling',
+    flex: 0.2,
+  },
+  {
+    field: 'question',
+    headerName: 'Opdracht',
+    flex: 0.2,
+  },
+  {
+    field: 'isCompleted',
+    headerName: 'Voltooid',
+    flex: 0.2,
+    renderCell: (params) =>
+      params.row.isCompleted ? <DoneRoundedIcon /> : <CloseRoundedIcon />,
+  },
+];
+
+const useStyles = makeStyles((theme) => ({
+  datagrid: {
+    marginTop: '1rem',
+    '& .MuiDataGrid-row:hover': {
+      cursor: 'pointer',
+    },
+  },
+}));
+
+const getRows = (questionGroups) => {
+  const rows = [];
+  questionGroups.forEach((gp) => {
+    const index = gp.index + 1;
```

Comment:
Zou je kunnen uitleggen wat `index` hier doet? ik snap het niet ;p

Reply:
dat is het opdrachtnummer, misschien beter "questionNumber" noemen?

Reply:
ah dus elke `row` die uit één questionGroup komt, moet dezelfde question/questionNumber hebben?

Reply:
yess